test/client/docs_test.rb

```diff
@@ -198,49 +192,49 @@
   end
 
   it "gets documents from the search index" do
-    h = @docs.get id: "1", type: "doc1"
+    h = $client.version_support.es_version_7_plus? ? @docs.get(id: "1") : @docs.get(id: "1", type: "book")
 
     refute_found h
 
     populate!
 
-    h = @docs.get id: "1", type: "doc1"
+    h = $client.version_support.es_version_7_plus? ? @docs.get(id: "1") : @docs.get(id: "1", type: "book")
 
     assert_found h
-    assert_equal "mojombo", h["_source"]["author"]
+    assert_equal "Author1", h["_source"]["author"]
   end
 
-  it "checks if documents exist in the search index" do
-    refute @docs.exists?(id: "1", type: "doc1")
+  it "checks if documents exist in the search indexx" do
```

Comment:
```suggestion
  it "checks if documents exist in the search index" do
```
Minor: removed extra `x`